gui/pdfmerger/pdfmerger.cpp

```diff
@@ -575,11 +602,17 @@ void PdfMerger::writeStreamAsXObject(PdfMergerPageInfo *pageInfo, Stream *stream
         for (int i=0; i<dict->getLength(); ++i)
         {
             Object value;
+#if POPPLER_VERSION < 5800
             dict->getVal(i, &value);
+#else
+            dict->getVal(i);
```

Comment:
@dtardon 
Is line ```dict->getVal(i);``` correct? We don't assign a result of getVal to a variable *value*.

Reply:
No, it's not correct.